content/projects/kauri/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Kauri Wallet"
+date: 2018-03-29T14:28:31+13:00
+lastmod: 2018-03-29T14:28:31+13:00
+draft: false
+author: "Paul Sanderson"
+description: "Kauri is an open-source, multi-currency wallet that will be the foundation for advanced features like mobile staking, merchant tools, atomic exchanges and many more great NavCoin features."
+resources: []
+categories: []
+filter: ["Wallets"]
+bg_image: "/images/icons/icon-wallet-purple-large.png"
+slug: ""
+type: "project"
+feature_image: ""
+show_title_text: false
+in_progress: true
+completed: false
+show_on_roadmap: true
+show_on_projects: true
+allow_click_through: true
+percent_complete: 10
+github_url: "https://github.com/Encrypt-S/kauri-wallet"
+project_url: ""
+reddit_url: ""
+twitter_url: ""
+card_color: "#6984D8"
+font_color: "#fff"
+weight: 0
+---
+{{< project_header
+    title="Kauri Wallet"
+    subhead=""
+    gradient=""
+    text="Kauri gives your crypto wallet the features you expect from your banking app like accounts and recurring payments but then so much more."
```

Comment:
I think we can improve this description:

"Kauri is the multi-currency crypto wallet you've been waiting for. Open source, fast, lightweight, secure and with all the features you'd expect from your bank and point-of-sales software. Kauri will set a new standard in managing your crypto assets'.

Something along these lines?